CHANGELOG.rst

```diff
@@ -27,6 +27,7 @@ Fixed
 -----
 - fixed missing ``tkinter`` dependency for running tests on Ubuntu
 - fixed issue with ``conversation`` JSON serialization
+- fixed random_seed does not work when using MultiProjectImporter
```

Comment:
With Python 3.6. this have worked before, it's more about keeping the order of the imports, right?
```suggestion
``MultiProjectImporter`` now imports files in the order of the import statements
```